app/assets/javascripts/alchemy/alchemy.spinner.js

```diff
@@ -0,0 +1,48 @@
+Alchemy.Spinner = function Spinner(size, styles) {
+  var html = '<div class="spinner">'+
+               '<svg width="100%" viewBox="0 0 28 28">'+
```

Comment:
Thinking about this, I should probably use a handlebars template for the spinner